marimo/_server/sessions.py

```diff
@@ -337,6 +348,22 @@ def close(self) -> None:
         self.kernel_manager.close_kernel()
         self.unsubscribe_consumer()
 
+    def instantiate(self, request: InstantiateRequest) -> None:
```

Comment:
this is much cleaner in here, instead of the extra roundabout inside the endpoint